lib/iris/fileformats/grib/__init__.py

```diff
@@ -962,7 +970,7 @@ def as_pairs(cube):
 def as_messages(cube):
     """
     Convert one or more cubes to GRIB messages.
-    Returns an iterable of grib_api GRIB messages.
+    Returns an iterable of grib_api GRIB message IDs.
```

Comment:
hmm, ok; this isn't going to be delivered by in this PR, we will ahve to wait until we are confident in message.GribMessage as a file writer, with appropriate grib save refactoring, before we can deliver this